R/use_lintr.R

```diff
@@ -0,0 +1,45 @@
+#' Use lintr in your project
+#'
+#' Create a minimal lintr config file as a starting point for customization
+#'
+#' @param path Path to project root, where a `.lintr` file should be created.
+#' If it already exists, an error will be thrown.
+#' @param type What kind of configuration to create?
+#'
+#'  * `minimal` creates a minimal lintr config, based off the default linters ([linters_with_defaults()]).
```

Comment:
"based on" or "based off of"

Reply:
nvm... "based on":

https://books.google.com/ngrams/graph?content=based+on+the%2C+based+off+the%2C+based+off+of+the&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cbased%20on%20the%3B%2Cc0%3B.t1%3B%2Cbased%20off%20the%3B%2Cc0%3B.t1%3B%2Cbased%20off%20of%20the%3B%2Cc0#t1%3B%2Cbased%20on%20the%3B%2Cc0%3B.t1%3B%2Cbased%20off%20the%3B%2Cc0%3B.t1%3B%2Cbased%20off%20of%20the%3B%2Cc0